producer/interceptor.go

```diff
@@ -54,7 +54,7 @@ func newTraceInterceptor(traceCfg *primitive.TraceConfig) primitive.Interceptor
 		beginT := time.Now()
 		err := next(ctx, req, reply)
 
-		producerCtx := primitive.GetProducerCtx(ctx)
+		producerCtx, _ := primitive.GetProducerCtx(ctx)
```

Comment:
what if producerCtx not exist, i see ignore exist return value here

Reply:
i will commit again later